src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const filteredMovies = moviesFromServer.filter(({ title, description }) => {
+    const fixedTitle = title.toLowerCase();
+    const fixedDescription = description.toLowerCase();
+    const fixedQuery = query.toLowerCase();
```

Comment:
[checklist point 2](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md)

You do the same in lines  9-10.